client/src/pages/Paste.js

```diff
@@ -20,6 +22,18 @@ const CreatedAt = styled(DateTime)`
 function Paste({ match }) {
   const [{ paste, error, loading }, doGet] = useGetPaste(match.params.pasteId);
 
+  function handleReport() {
+    const reportIcon = '🚨REPORT🚨';
+    const pasteLink = `${window.location.origin}/${paste._id}`;
+
+    const slackReport = {
+      text: `${reportIcon} \n===\n${paste.author}: \n"${paste.value}"\n=== \nID: ${paste._id} \n${pasteLink} \n${reportIcon}`
+    };
+
+    alert('Report 🚨');
+    postPasteToSlack(slackReport);
+  }
+
```

Comment:
I think this is something that can be moved to the ReportButton component so the Paste.js is kept cleaner and the handle report functionality in same file that holds the component.